components/pages/Home/Home.tsx

```diff
@@ -10,19 +11,34 @@ import {
   Footer,
 } from "components";
 import styled from "styled-components";
+import { useIntersectionObserver } from "hooks";
 
 export function Home() {
+  const headerRef = useRef<HTMLDivElement | null>(null);
+  const howItWorksRef = useRef<HTMLDivElement | null>(null);
+  const eHeader = useIntersectionObserver(headerRef, {
+    threshold: 0.1328,
+  });
+  const isIntersectingLightArea = !!eHeader?.isIntersecting;
+  const eHotItWorks = useIntersectionObserver(howItWorksRef, {
+    threshold: 0.49,
+  });
```

Comment:
These thresholds seem to give the desired effect but seem very arbitrary, I'd rather it be cleaner.